assets/components/doubleHeading/doubleHeading.jsx

```diff
@@ -1,11 +1,21 @@
+// @flow
+
 // ----- Imports ----- //
 
 import React from 'react';
 
 
+// ---- Types ----- //
+
+type PropTypes = {
+  heading: string,
+  subheading: ?string,
```

Comment:
If we setup subheading as a Maybe Type of string, it means that we allow doubleHeading elements with just one heading. In that case, what would be the difference between this component and a `<h1>`? Should it be the type of this parameter just string so the component makes sense?